doc/pages/user/017-astarte_grafana_plugin.md

```diff
@@ -0,0 +1,49 @@
+# Astarte Datasource Plugin for Grafana
+
+Astarte Datasource Plugin conveys data from Astarte to
+[Grafana](https://github.com/grafana/grafana), the open-source platform for
+monitoring and observability, developed by [Grafana Labs](https://grafana.com/).
+Actual data visualisation is provided by Grafana via its
```

Comment:
`visualisation` -> `visualization`

Reply:
Maybe `provided by Grafana via its plugins` -> `provided by Grafana via its chart panels`, or something similar, to convey the difference in regards to our Datasource plugin.